### ISSUE TITLE
Column is not updated when columns are moved.

### ISSUE THREAD

**Issue by KyGuyMayor** [jeffreydwalter/ColReorderWithResize#18]
When I move a column, the colReorder.order() function still returns the indexes in the default order.

**Comment by jeffreydwalter** [comment]
Are you calling order() yourself? Can you provide a piece of code that reproduces the issue?

The `order()` function takes two arguments: https://github.com/jeffreydwalter/ColReorderWithResize/blob/master/ColReorderWithResize.js#L1739-L1749

**Comment by KyGuyMayor** [comment]
perhaps I have a misunderstanding of the function. Is there a way to return the current column order after a user has re-ordered columns?

**Comment by jeffreydwalter** [comment]
@KyGuyMayor nah, you're right. I don't really use this library anymore, so I'm a bit rusty on the API.

**Comment by jeffreydwalter** [comment]
If you could provide a work (not working) example, I'd be happy to debug it for you (or I will happily accept a pull request that fixes the issue).

**Comment by vmarinaro** [comment]
Hi, I have try to fix it in this way. In Move the internal array elements after Colums options regenerate I inserted this code.
  ```
 /* regenerate event handlers for the drag and drop */
    for ( i=0, iLen=iCols ; i<iLen ; i++ )
    {
        if ( i > oSettings._colReorder.s.fixed-1 && i < iLen - oSettings._colReorder.s.fixedRight )
        {           
            oSettings._colReorder._fnMouseListener( i, oSettings.aoColumns[i].nTh );
        }       
    }
```

**Comment by jeffreydwalter** [comment]
@KyGuyMayor does this fix the problem? If so, please make a PR and I'll merge in your changes.

**Comment by KyGuyMayor** [comment]
Sorry for the super late reply. I no longer have access to the project that was using this due to a job change. So I am unable to properly test it. I am okay with this being closed.
